plugins/woocommerce/changelog/add-merge-playwright

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I think as the tests aren't included in the WC zip we can use `patch` significance, make a comment, and leave the changelog entry line blank. [https://packagist.org/packages/automattic/jetpack-changelogger#user-content-changelog-entry](https://href.li/?https://packagist.org/packages/automattic/jetpack-changelogger#user-content-changelog-entry)

Reply:
@tammullen I made that update, but it reset the approval status of the PR.  Would you mind giving it an approval so I can merge?